mimic/test/test_nova.py

```diff
@@ -1374,12 +1374,13 @@ def use_creation_behavior(self, name, parameters, criteria):
         criterion = {"name": name,
                      "parameters": parameters,
                      "criteria": criteria}
-        set_criteria = request(self, self.root, "POST",
-                               self.nova_control_endpoint +
-                               "/behaviors/creation/",
-                               json.dumps(criterion))
-        set_criteria_response = self.successResultOf(set_criteria)
-        self.assertEqual(set_criteria_response.code, 201)
+        set_criteria = json_request(self, self.root, "POST",
+                                    self.nova_control_endpoint +
+                                    "/behaviors/creation/",
+                                    json.dumps(criterion))
+        response, body = self.successResultOf(set_criteria)
+        self.assertEqual(response.code, 201)
+        self.assertTrue(body.get("id"))
```

Comment:
`assertTrue` is just a *tad* too weak of an assertion for the behavior that the docstring of the method alleges to add.  I'd do something like this:

```python
behavior_id = body.get("id")
self.assertIsInstance(behavior_id, text_type)
parsed = uuid.UUID(hex=behavior_id)
self.assertEquals(parsed.version, 4)
```

I also assume clients are going to want to do something with the behavior ID, although it's fine to leave that for a future PR.